app/graphql/types/mutation_type.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+module Types
+  class MutationType < Types::BaseObject
+    # TODO: remove me
```

Comment:
🤔

Reply:
will be fixed by https://github.com/espoo-dev/rails_boilerplate/issues/146